Document changes to the sign-in experience (#1100)
spectrocloud/librarium: content/docs/08-user-management/1-user-authentication.md

* [API Key](/user-management/user-authentication/#api-key)

* [Authorization Token](/user-management/user-authentication/#authorization-token)

The API key and the authorization token method can be used when interacting with Palette REST APIs for automation and programmatic purposes.

# UI Authentication
<!-- vale off -->
You can log into Palette by visiting the Palette at [http://console.spectrocloud.com](https://console.spectrocloud.com). If you are a user of a Palette Enterprise instance, then you should use the URL provided by your Palette system 
administrator, such as `example-company.console.spectrocloud.com.` 
<!-- vale on -->
## Account Sign Up

You can sign up for a Palette SaaS account by visiting [Palette](https://console.spectrocloud.com) or an Enterprise Palette account under your organization by using your organization's custom Palette URL.

When you create an account, you can create a username and password or create the account through a third-party identity provider, GitHub, Google, or other OIDC providers that are enabled for your organization. For Palette SaaS, GitHub and Google are automatically enabled for SSO integration.

## Sign In Flow

> Starting with Palette 3.2, the user sign-in flow can be different depending on how you created your Palette account. If you made your user with a username and password, then you may be promoted to select the organization you wish to log in to. If you are a member of a single organization, then you will not be prompted for an organization selection.

Reviewer: ```suggestion
Starting with Palette 3.2, the user sign-in flow can be different depending on how you created your Palette account. If you created your user with a username and password, then you may be promoted to select the organization you wish to log in to. If you are a member of a single organization, then you will not be prompted for an organization selection.
```

> Starting with Palette 3.2, the user sign-in flow can be different depending on how you created your Palette account. If you created your user with a username and password, then you may be promoted to select the organization you wish to log in to. If you are a member of a single organization, then you will not be prompted for an organization selection.

Author: ```suggestion
Starting with Palette 3.2, the user sign-in flow can be different depending on how you created your Palette account. If you created your user with a username and password, then you may be prompted to select the organization you wish to log in to. If you are a member of a single organization, then you will not be prompted for an organization selection.
```

If you created an account through SSO and are a member of different organizations, then you must first select the organization name you wish to log in to. Click on the **Sign in to your organization** button for the option to specify the organization name. If you need help remembering the organization name, click on the **Forgot your organization name?** button and provide your email address to receive an email containing your organization name and its login URL.

<br />

<InfoBox>

> If you are a Palette Enterprise user, use the custom Palette URL for an optimized log in experience and avoid specifying the organization name. 

Reviewer: ```suggestion
If you are a Palette Enterprise user, use the custom Palette URL for an optimized login experience and avoid specifying the organization name. 
```
